src/Npgsql/Internal/TypeInfoMapping.cs

```diff
@@ -43,16 +48,30 @@ public TypeInfoMapping(Type type, DataTypeName dataTypeName, bool isDefault, Typ
         // Not a match
         return null;
     }
+
+    string DebuggerDisplay
```

Comment:
Yeah I'm now big on DebuggerDisplay everywhere :)

Reply:
BTW I'm not sure, but is it possible that ToStrings can't be trimmed because they're virtual from object (I remember we had a problem like that with the IEnumerator implementation of NpgsqlDataReader)? If so that could be a reason to avoid them (DebuggDisplay obviously doesn't have that problem)